module5/lessons/Week1/CoffeeChatKickoff.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Coffee Chat Kickoff
+---
+
+In Mod 4, you all did a great job with your first coffee chat! I know for some of you it was quite stressful, my first couple coffee chats definitely were! I hope you feel proud of yourself for breaking the ice and are now ready to keep this energy moving forward 🚀
+
+As we talked about in the CoffeeChatPD lesson there are many benefits of coffee chats including:
+
+**Real-world Context:** Coffee chats help bridge the gap between theoretical knowledge gained in class and real-world applications. Interviewees will share real world examples that illustrate how coding skills are used in actual projects and workflows.
+
+**Networking:** Engaging in coffee chats allows you to expand your professional network. Building connections with experienced professionals can lead to mentorship opportunities, potential job referrals, and a broader understanding of the industry landscape. But it’s key to note, the tone of the coffee chat should not be job seeking, even though there is a fair chance it might help lead to a job.
+
+**Interview Preparation:** While you are not the one interviewed in this context, coffee chats still develop your comfort level talking with an unknown professional and help the actual interviews feel less scary.
+
+## The Assignment
+
+The expectation for Mod 5 is that **you will have two more coffee chats by the end of Week 4**. 
+
+✅Deliverable✅: By the end of Week 4, send the LinkedIns of the two people you chatted with along with your answers to the following reflection questions:

```

Comment:
Definitely interested in your thoughts on this timeline. I do think we want 2 coffee chats, I was torn if I should make these two separate deliverables because I know a four week timeline is long and it's easy to procrastinate. But that seemed logistically difficult. 🤷‍♀️

Reply:
I think that a longer timeline is ok - I might add a note that students should schedule reminders throughout the inning.  Like, a slack reminder if 2 days to reach back out if they haven't heard back yet... this could help them practice long-term planning skills!

Reply:
Ooo I like this idea!